frontend/online-shop/src/index.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import ReactDOM from "react-dom";
+import "./index.css";
+import App from "./App";
+import reportWebVitals from "./reportWebVitals";
+import { ThemeProvider } from "@material-ui/core";
+import theme from "./theme";
+
+ReactDOM.render(
+  <React.StrictMode>
+    <ThemeProvider theme={theme}>
```

Comment:
It should be better to move the `ThemeProvider` wrapper within the `App` component so this `index.tsx` can be _without any logic_.

Reply:
Also under `ThemeProvider` you can place a `CssBaseline` wrapper that will make a css and material properties reset. You can read more about this on their official page: [CssBaseline](https://material-ui.com/components/css-baseline/)